api/auth/index.ts

```diff
@@ -94,10 +96,10 @@ export async function oauth(req: EnhancedRequest, res: express.Response, next: e
             res.cookie(config.cookies.token, accessToken)
 
             const orgIdResponse = await getOrganisationId(details)
-            const orgId = orgIdResponse.data.id
-
+            // const orgId = orgIdResponse.data.id
+            //
             session.auth = {
-                orgId,
+                orgId: orgIdResponse.data.id,
```

Comment:
why did you change this?  use one or another

Reply:
To keep it in object the Key: Reference , as you had it outside as const.